ReactAndroid/src/main/java/com/facebook/react/views/scroll/ReactScrollViewCommandHelper.java

```diff
@@ -64,6 +77,11 @@
         viewManager.scrollTo(scrollView, new ScrollToCommandData(destX, destY, animated));
         return;
```

Comment:
Actually we need a `break` or `return` statement here, otherwise control flow will fall-through: https://docs.oracle.com/javase/tutorial/java/nutsandbolts/switch.html